src/test/java/org/cactoos/io/InputOfTest.java

```diff
@@ -240,36 +240,42 @@ public void readsInputIntoBytes() throws IOException {
 
     @Test
     public void readsStringBuilder() throws IOException {
+        final String starts = "Name it, ";
+        final String ends = "then it exists!";
         MatcherAssert.assertThat(
-            "Can't receive string builder",
+            "Can't receive a string builder",
             new String(
                 new InputAsBytes(
                     new InputOf(
-                        new StringBuilder("Name it, then it exists!")
+                        new StringBuilder(starts)
+                            .append(ends)
```

Comment:
Yes it's a good idea but here in the test, .append is just to illustrate the use of InputOf in context.